--- elasticsearch/src/main/java/org/apache/zeppelin/elasticsearch/client/HttpBasedClient.java.orig    2023-03-30 08:23:16.846106159 +0000
+++ elasticsearch/src/main/java/org/apache/zeppelin/elasticsearch/client/HttpBasedClient.java    2023-03-30 08:24:48.403187991 +0000
@@ -168,7 +168,7 @@
               .succeeded(true)
               .hit(new HitWrapper(
                   getFieldAsString(body, "_index"),
-                  getFieldAsString(body, "_type"),
+                  "unknown",
                   getFieldAsString(body, "_id"),
                   getFieldAsString(body, "_source")));
         } else {
@@ -178,7 +178,7 @@
               .succeeded(true)
               .hit(new HitWrapper(
                   hit.getString("_index"),
-                  hit.getString("_type"),
+                  "unknown",
                   hit.getString("_id"),
                   hit.opt("_source").toString()));
         }
@@ -214,7 +214,7 @@
             .succeeded(true)
             .hit(new HitWrapper(
                 getFieldAsString(body, "_index"),
-                getFieldAsString(body, "_type"),
+                "unknown",
                 getFieldAsString(body, "_id"),
                 null));
       } else {
@@ -252,7 +252,7 @@
             .succeeded(true)
             .hit(new HitWrapper(
                 getFieldAsString(result, "_index"),
-                getFieldAsString(result, "_type"),
+                "unknown",
                 getFieldAsString(result, "_id"),
                 null));
       } else {
@@ -332,7 +332,7 @@
                 hit.opt("_source") != null ? hit.opt("_source") : hit.opt("fields");
             response.addHit(new HitWrapper(
                 hit.getString("_index"),
-                hit.getString("_type"),
+                "unknown",
                 hit.getString("_id"),
                 data.toString()));
           }
